package.json

```diff
@@ -5,11 +5,11 @@
   "main": "lib/index.js",
   "scripts": {
     "build": "rimraf lib && babel src --out-dir lib",
-    "prepublish": "npm run lint && npm run test && npm run build",
+    "prepublish": "in-install || (npm run lint && npm run test && npm run build)",
     "coveralls": "cat ./reports/lcov.info | coveralls",
     "lint": "eslint  --config .eslintrc .",
     "lint:fix": "npm run lint -- --fix",
-    "pretest": "npm run lint",
+    "pretest": "check-node-version --node '>= 6' && npm run lint",
```

Comment:
you actually want this to be `check-node-version --node '< 6' || npm run lint`, otherwise it will fail on < 6.

however, this should be done with travis.yml imo and not in package.json.